### ISSUE TITLE
Color contrast for chart text

### ISSUE THREAD

**Issue by SaptakS** [freedomofpress/pressfreedomtracker.us#1325]
The text on top of the charts have a color that doesn't meet the color contrast needed for WCAG AA
